lm_eval/tasks/paloma/paloma.yaml

```diff
@@ -0,0 +1,22 @@
+group:
+  - paloma
+dataset_path: allenai/paloma
+output_type: loglikelihood_rolling
+validation_split: val
+test_split: test
+doc_to_text: ""
+doc_to_target: !function paloma_utils.doc_to_target
```

Comment:
This can directly refer to the feature.
```
doc_to_target: text
```

Reply:
I tried that option before, but there are some items in the `paloma_dolma_100_programing_languages` task of the form: `['test', 'test2', 'test3' ]` which get interpredted as lists instead of strings, and cause an exception to be raised when the evaluation script tries to tokenize it. 

I tried placing `doc_to_target: {{ text|string }}`, but the same thing still occurred. 

If you have any other suggestion, let me know